controller-server/src/main/java/com/yahoo/vespa/hosted/controller/versions/VersionStatus.java

```diff
@@ -262,4 +265,14 @@ private static VespaVersion createVersion(DeploymentStatistics statistics,
                                 confidence);
     }
 
+    /** Whether no version on a newer major, with normal or higher confidence, can be deployed. */
+    public boolean isOnCurrentMajor(Version version) {
+        for (VespaVersion available : deployableVersions())
+            if (   available.confidence().equalOrHigherThan(Confidence.high)
```

Comment:
Shouldn't this be `equalOrHigherThan(Confidence.normal)`, or is the javadoc wrong?

Reply:
Wrong javadoc, thanks.